app/views/layouts/application.html.erb

```diff
@@ -4,11 +4,21 @@
     <title>Evensteven</title>
     <%= csrf_meta_tags %>
 
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" %>
```

Comment:
Again, servig assets over CDN is not a practice we encourage. Also, in case of Bootstrap, you need the js files and have jQuery installed.

Reply:
I dont really understand. You guys told us to use CDN's and we are not using Js or JQuery at this point°

Reply:
@tochman

Reply:
When and who told you that? Also, I'm telling you a different thing now. Also, Bootstrap simply won't work without the js dependencies and jQuery.

Reply:
Well we did a minihomework about why it's good and why we should use it!
But just to clarify, we should download the bootstrap lib to our project?!

Reply:
There is probably a gem for that. Like for most other things...

Reply:
Thank you @tochman